smartmodules/json-sql/SmartModule.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "json-sql"
+group = "infinyon"
+version = "0.1.0"
+apiVersion = "0.1.0"
+description = "JSON to SQL transformation Smartmodule"
+license = "Apache-2.0"
+repository = "https://github.com/infinyon/fluvio"
+
+
+[params]
+with = { description = "Transformation mapping" }
```

Comment:
shouldn't this be something like `mapping`?

Reply:
I agree that `mapping` now feels more descriptive, but the change will also affect the SQL connector. Should we do it here?

Reply:
Probably both

Reply:
Done